README.md

```diff
@@ -156,6 +173,13 @@ You have the ability to customize the way things are placed on the board (maybe
 ### Custom Colors
 You have the ability to customize the colors of everything on the board. See the `ledcolors/` directory for more information.
 
+### Weather
+This scoreboard will use a weather API to gather weather information at various times. This information is displayed on your teams offdays for your area and also displayed during each game's pregame information. The weather API we use is from OpenWeatherMaps. OpenWeatherMaps API requires an API key to fetch this data so you will need to take a quick minute to sign up for an account and copy your own API key into your `config.json`.
```

Comment:
We should add a step for this in the installation section since I'm assuming this is required to get the board to run.

Reply:
I added a line/link to the weather section in the installation section. Not having a valid API key fails gracefully and will give a warning in the terminal (it just won't display on the screen if it fails). Let me know if you think we need more here.